### PR TITLE
Configuration article

### PR DESCRIPTION
**Pull request by cromedome**
An article on how configuration of Dancer2 apps has changed in 2016, with some examples on how to use it.

### CODE REVIEW

File: pending/configuring-your-dancer-apps.pod [hunk 1]
```diff
@@ -0,0 +1,162 @@
+=head1 Configuring Your Dancer Applications
+
+It's been a really exciting year for Dancer2, and one of the many things
```
Comment (xsawyerx) [inline pending/configuring-your-dancer-apps.pod]: Exciting two years since last advent calendar.

File: pending/configuring-your-dancer-apps.pod [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+=head2 Wrapping Up
+
+Dancer2 developers gained a lot of flexibility in how to to configure their 
+applications in 2016. The core team would like to give a shout-out to Jonathan
+Scott Duff (aka perlpilot) for his work in making this possible.
+
+=head2 Author
+
+Jason A. Crome (CromeDome) C<cromedome@cpan.org>
```
Comment (xsawyerx) [inline pending/configuring-your-dancer-apps.pod]: Let's add a copyright (@syspete and I waver our copyright with a small note, but feel free to use any other license you wish.)

**Comment by xsawyerx** [comment]
@cromedome Can we merge this?

**Comment by cromedome** [comment]
Go for it!

**Comment by xsawyerx** [comment]
Merged, thanks! :+1: 
Due date: 17th.

**Pull request closed without merging**